.vscode/launch.json

```diff
@@ -15,6 +16,33 @@
                 "crawl", 
                 "tricity"
             ]
+        }, 
+        {
+            "name": "Python: flats",
+            "type": "python",
+            "request": "launch",
+            "module": "scrapy",
+            "justMyCode": true,
+            "cwd": "${workspaceFolder}/flats/flatsscraper/flatsscraper",
```

Comment:
Jak zmienisz lokalizację spidera flatscrapper to tutaj ścieżka też trzbea będzie zmienić

Reply:
ok